docs/getting-started/3rd-party-bindings.md

```diff
@@ -0,0 +1,18 @@
+# 3rd Party Bindings
+
+## WASM
+
+<a href="https://github.com/MetaMask/bdk-wasm" target="_blank">bdk-wasm</a>
+
+Note: The bdk-wasm lib can be used in just about any modern JavaScript environment, including Browser, Node, ReactNative, etc.
+
+## Flutter
+
+<a href="https://github.com/LtbLightning/bdk-flutter" target="_blank">bdk-flutter</a>
+
+
+## ReactNative
+
+<a href="https://github.com/xsats/react-native-bdk" target="_blank">react-native-bdk</a>
```

Comment:
The bdk-rn library is [this one](https://github.com/LtbLightning/bdk-rn)! Sorry we never really made that clear.